fuse/gocryptfs/Portfile

```diff
@@ -0,0 +1,143 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           golang 1.0
+PortGroup           fuse 1.0
+
+go.setup            github.com/rfjakob/gocryptfs v2.0.1
```

Comment:
Move the `v` to the third argument. Then this prefix will not be included in the port version.
```
go.setup            github.com/rfjakob/gocryptfs 2.0.1 v
```